common/config/babel/.babelrc.js

```diff
@@ -16,27 +17,28 @@ process.env['COMMUNICATION_REACT_FLAVOR'] === 'stable' &&
         // API for injecting custom buttons in he control bar for
         // `CallComposite` and `CallWithChatComposite`.
         'control-bar-button-injection',
+        // props to allow Contoso to overwrite timestamp format for chat messages, one in locale and one in message thread component
+        'date-time-customization',
         // Demo feature. Used in live-documentation of conditional compilation.
         // Do not use in production code.
         'demo',
+        // dialpad
+        'dialpad',
         // Ability to upload/download files in message thread.
         'file-sharing',
-        // Adhoc calls to a Teams user.
-        'teams-adhoc-call',
-        // PSTN calls 
-        'PSTN-calls',
-        // props to allow Contoso to overwrite timestamp format for chat messages, one in locale and one in message thread component
-        'date-time-customization',
         // 1 to N Calling feature.
         'one-to-n-calling',
-        // Support Calling SDK isReceiving flag, shows a loading spinner on the video tile when isAvailable is true but isReceiving is false
-        'video-stream-is-receiving-flag',
-        // dialpad
-        'dialpad',
+        // PSTN calls 
+        'PSTN-calls',
         // rooms
         'rooms',
-        'call-readiness',
-        'unsupported-browser'
+        // Adhoc calls to a Teams user.
+        'teams-adhoc-call',
+        // Joining calls using teams token (CTE)
+        'teams-call',
```

Comment:
nit: teams-call is ambiguous as a feature name (we can currently make teams calls, just not as a Teams identity)
```suggestion
        'teams-identity-support',
```
(or whatever the official standin name for CTE is)